2024-03-13 16:07:43,689 INFO    StreamThr :27916 [internal.py:wandb_internal():86] W&B internal server running at pid: 27916, started at: 2024-03-13 16:07:43.688436
2024-03-13 16:07:43,690 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status
2024-03-13 16:07:43,694 INFO    WriterThread:27916 [datastore.py:open_for_write():85] open: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\run-b3ojb8wd.wandb
2024-03-13 16:07:43,696 DEBUG   SenderThread:27916 [sender.py:send():380] send: header
2024-03-13 16:07:43,696 DEBUG   SenderThread:27916 [sender.py:send():380] send: run
2024-03-13 16:07:44,699 INFO    SenderThread:27916 [dir_watcher.py:__init__():211] watching files in: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files
2024-03-13 16:07:44,699 INFO    SenderThread:27916 [sender.py:_start_run_threads():1122] run started: b3ojb8wd with start time 1710317263.688435
2024-03-13 16:07:44,717 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: check_version
2024-03-13 16:07:44,717 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: check_version
2024-03-13 16:07:49,722 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:07:49,736 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: run_start
2024-03-13 16:07:49,778 DEBUG   HandlerThread:27916 [system_info.py:__init__():32] System info init
2024-03-13 16:07:49,778 DEBUG   HandlerThread:27916 [system_info.py:__init__():47] System info init done
2024-03-13 16:07:49,778 INFO    HandlerThread:27916 [system_monitor.py:start():181] Starting system monitor
2024-03-13 16:07:49,779 INFO    SystemMonitor:27916 [system_monitor.py:_start():145] Starting system asset monitoring threads
2024-03-13 16:07:49,779 INFO    HandlerThread:27916 [system_monitor.py:probe():201] Collecting system info
2024-03-13 16:07:49,784 INFO    SystemMonitor:27916 [interfaces.py:start():190] Started cpu monitoring
2024-03-13 16:07:49,784 INFO    SystemMonitor:27916 [interfaces.py:start():190] Started disk monitoring
2024-03-13 16:07:49,785 INFO    SystemMonitor:27916 [interfaces.py:start():190] Started gpu monitoring
2024-03-13 16:07:49,786 INFO    SystemMonitor:27916 [interfaces.py:start():190] Started memory monitoring
2024-03-13 16:07:49,813 INFO    SystemMonitor:27916 [interfaces.py:start():190] Started network monitoring
2024-03-13 16:07:49,822 DEBUG   HandlerThread:27916 [system_info.py:probe():196] Probing system
2024-03-13 16:07:49,822 DEBUG   HandlerThread:27916 [system_info.py:probe():244] Probing system done
2024-03-13 16:07:49,822 DEBUG   HandlerThread:27916 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.22000-SP0', 'python': '3.10.9', 'heartbeatAt': '2024-03-13T08:07:49.822880', 'startedAt': '2024-03-13T08:07:43.674432', 'docker': None, 'cuda': None, 'args': ('--task=P', '--weather=rainy'), 'state': 'running', 'program': 'E:\\自动驾驶\\车位检测\\parking-space-Identification-in-different-weather-main\\train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'host': 'LAPTOP-QIVT4NMT', 'username': 'lenovo', 'executable': 'E:\\anaconda\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 3201.0, 'min': 0.0, 'max': 3201.0}, 'cpu_freq_per_core': [{'current': 3201.0, 'min': 0.0, 'max': 3201.0}], 'disk': {'/': {'total': 465.74511337280273, 'used': 176.2876091003418}}, 'gpu': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'memory_total': 6442450944}], 'memory': {'total': 15.855525970458984}}
2024-03-13 16:07:49,822 INFO    HandlerThread:27916 [system_monitor.py:probe():211] Finished collecting system info
2024-03-13 16:07:49,822 INFO    HandlerThread:27916 [system_monitor.py:probe():214] Publishing system info
2024-03-13 16:07:49,823 DEBUG   HandlerThread:27916 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-03-13 16:07:49,823 DEBUG   HandlerThread:27916 [system_info.py:_save_pip():68] Saving pip packages done
2024-03-13 16:07:49,824 DEBUG   HandlerThread:27916 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-03-13 16:07:49,834 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:07:50,726 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\requirements.txt
2024-03-13 16:07:50,726 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\conda-environment.yaml
2024-03-13 16:07:51,880 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:07:53,905 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:07:55,940 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:07:56,766 DEBUG   HandlerThread:27916 [system_info.py:_save_conda():87] Saving conda packages done
2024-03-13 16:07:56,767 INFO    HandlerThread:27916 [system_monitor.py:probe():216] Finished publishing system info
2024-03-13 16:07:56,779 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\conda-environment.yaml
2024-03-13 16:07:56,779 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-metadata.json
2024-03-13 16:07:56,787 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:07:56,787 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: keepalive
2024-03-13 16:07:56,788 DEBUG   SenderThread:27916 [sender.py:send():380] send: files
2024-03-13 16:07:56,788 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2024-03-13 16:07:56,800 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 16:07:56,800 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: stop_status
2024-03-13 16:07:56,801 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 16:07:57,572 DEBUG   SenderThread:27916 [sender.py:send():380] send: telemetry
2024-03-13 16:07:57,793 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:07:57,977 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:07:59,812 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:00,005 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:00,596 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:02,047 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:03,371 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:03,373 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:03,373 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:03,374 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:03,829 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:04,080 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:05,822 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:05,822 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:05,823 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:05,824 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:05,825 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:05,825 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:05,826 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:05,826 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:05,827 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:05,849 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:05,850 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:06,117 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:07,854 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:08,153 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:08,596 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:08,597 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:08,597 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:08,598 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:08,855 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:09,871 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:10,196 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:10,659 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:10,659 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:10,660 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:10,661 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:10,661 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:10,662 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:10,662 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:10,663 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:10,882 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:11,674 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:11,803 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 16:08:11,818 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 16:08:11,819 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: stop_status
2024-03-13 16:08:11,884 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:12,228 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:13,462 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:13,462 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:13,463 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:13,463 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:13,905 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:14,267 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:15,489 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:15,491 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:15,491 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:15,492 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:15,493 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:15,493 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:15,493 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:15,494 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:15,862 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C71C1F0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:08:15,862 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 16:08:15,862 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:08:15,927 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:15,928 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:16,306 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:17,511 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:17,959 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\config.yaml
2024-03-13 16:08:17,959 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:18,283 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:18,284 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:18,284 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:18,285 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:18,335 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:18,968 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:19,977 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:20,361 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:20,362 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:20,363 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:20,363 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:20,370 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:20,371 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:20,371 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:20,380 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:20,424 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:20,990 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:22,007 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:22,454 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:23,103 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:23,104 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:23,104 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:23,105 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:23,105 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:24,031 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:24,032 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:24,475 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:25,078 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:25,079 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:25,080 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:25,080 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:25,081 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:25,081 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:25,081 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:25,082 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:26,056 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:26,057 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:26,512 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:26,817 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 16:08:26,817 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 16:08:26,818 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: stop_status
2024-03-13 16:08:27,847 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:27,848 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:27,848 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:27,849 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:28,088 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:28,546 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:28,850 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:29,833 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:29,834 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:29,835 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:29,835 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:29,836 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:29,836 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:29,836 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:29,837 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:29,877 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C7719C0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:08:29,877 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 16:08:29,877 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:08:30,092 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:30,093 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:30,592 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:32,099 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:32,606 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:32,608 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:32,608 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:32,609 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:32,623 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:33,106 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:34,108 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:34,645 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:34,647 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:34,647 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:34,648 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:34,648 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:34,649 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:34,650 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:34,650 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:34,650 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:34,666 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:35,123 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:36,139 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:36,709 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:37,426 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:37,427 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:37,427 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:37,428 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:38,164 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:38,165 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:38,737 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:39,403 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:39,404 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:39,404 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:39,405 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:39,406 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:39,406 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:39,406 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:39,407 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:40,179 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:40,180 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:40,415 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:40,772 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:41,827 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 16:08:41,827 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 16:08:41,827 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: stop_status
2024-03-13 16:08:42,165 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:42,165 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:42,166 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:42,167 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:42,206 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:42,809 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:44,119 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:44,120 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:44,121 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:44,121 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:44,122 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:44,122 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:44,122 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:44,123 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:44,238 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:44,239 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:44,841 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:46,139 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:46,268 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:46,868 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:46,869 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:46,869 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:46,870 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:46,888 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:47,281 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:48,289 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:48,912 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:48,912 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:48,913 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:48,913 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:48,914 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:48,915 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:48,915 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:48,916 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:48,921 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:49,300 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:49,820 DEBUG   SystemMonitor:27916 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2024-03-13 16:08:49,821 DEBUG   SenderThread:27916 [sender.py:send():380] send: stats
2024-03-13 16:08:50,316 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:50,873 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002240C930430>, 'Connection to storage.googleapis.com timed out. (connect timeout=None)'))
2024-03-13 16:08:50,873 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 16:08:50,873 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:08:50,873 INFO    wandb-upload_0:27916 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 95, in create_connection
    raise err
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 85, in create_connection
    sock.connect(sa)
TimeoutError: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 714, in urlopen
    httplib_response = self._make_request(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 403, in _make_request
    self._validate_conn(conn)
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 1053, in _validate_conn
    conn.connect()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 179, in _new_conn
    raise ConnectTimeoutError(
urllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPSConnection object at 0x000002240C930430>, 'Connection to storage.googleapis.com timed out. (connect timeout=None)')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 487, in send
    resp = conn.urlopen(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 798, in urlopen
    retries = retries.increment(
  File "E:\anaconda\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002240C930430>, 'Connection to storage.googleapis.com timed out. (connect timeout=None)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 508, in send
    raise ConnectTimeout(e, request=request)
requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002240C930430>, 'Connection to storage.googleapis.com timed out. (connect timeout=None)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2580, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 508, in send
    raise ConnectTimeout(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-03-13 16:08:51,013 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:51,664 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:51,664 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:51,665 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:51,665 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:51,666 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:52,326 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:53,046 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:53,614 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:53,614 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:53,615 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:53,615 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:53,617 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:53,617 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:53,617 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:53,618 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:54,332 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:54,333 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:55,077 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:56,364 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:56,380 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:56,381 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:56,381 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:56,382 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:56,842 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 16:08:56,842 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 16:08:56,843 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:08:56,843 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: stop_status
2024-03-13 16:08:57,103 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:57,368 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:08:58,369 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:08:58,419 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:58,419 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:08:58,420 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:58,420 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:58,421 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:58,422 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:08:58,422 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:08:58,422 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:08:59,145 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:08:59,373 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:00,376 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:01,181 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:01,252 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:01,253 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:01,253 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:01,254 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:01,379 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:02,261 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:02,385 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:03,205 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:03,308 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:03,309 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:03,310 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:03,310 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:03,311 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:03,311 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:03,311 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:03,312 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:03,392 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:04,404 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:05,225 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:06,126 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:06,127 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:06,127 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:06,128 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:06,428 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:07,264 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:07,814 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C932D70>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:09:07,814 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 16:09:07,814 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:09:08,068 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:08,069 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:08,069 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:08,070 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:08,070 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:08,071 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:08,072 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:08,072 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:08,073 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:08,453 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:08,454 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:09,295 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:10,475 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:10,837 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:10,838 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:10,838 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:10,839 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:11,326 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:11,489 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:11,838 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 16:09:11,838 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: stop_status
2024-03-13 16:09:11,854 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 16:09:12,505 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:12,891 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:12,891 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:12,892 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:12,893 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:12,894 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:12,894 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:12,894 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:12,895 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:13,358 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:13,507 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:13,904 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:14,523 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:15,390 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:15,649 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:15,650 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:15,650 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:15,651 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:16,549 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:17,420 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:17,614 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:17,615 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:17,615 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:17,616 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:17,616 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:17,617 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:17,617 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:17,618 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:18,566 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:18,567 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:19,458 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:19,632 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:19,823 DEBUG   SenderThread:27916 [sender.py:send():380] send: stats
2024-03-13 16:09:20,379 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:20,379 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:20,380 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:20,380 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:20,571 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:20,572 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:21,550 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:22,465 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:22,466 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:22,467 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:22,467 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:22,468 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:22,469 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:22,469 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:22,470 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:22,602 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:22,602 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:23,572 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:24,606 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:25,270 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:25,270 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:25,271 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:25,271 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:25,272 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:25,595 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:25,609 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:26,614 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:26,851 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 16:09:26,852 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: stop_status
2024-03-13 16:09:26,868 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 16:09:27,265 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:27,266 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:27,266 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:27,267 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:27,267 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:27,268 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:27,268 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:27,269 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:27,621 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:27,625 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:28,636 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:28,805 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-metadata.json?Expires=1710403680&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=D5xyEDhT1BSflFpOE30eEnagEiAn7QqQiSn3rZV0FjltyJdnzNGBSdl4WgrrO4kFYUi2i3fCnTF9IEBBix16GjuBXbM%2BuPxWmWwgg%2BVPyzHOL7XFT3dWr0rLZi1FCIK8OHuuWfzFEsqEaIDrSaoEqlPXBsEJbnc22ZJa7wFw%2FL3skD8UM%2BlKXQcMrB0nD5gXIQVtss9suoeRWyGr5Xb6eSIpHskgrf6762g%2Bd0T%2BSyJNz3bA%2Br1OFF16puG9IyDctKJqT6HQ3M3NLQXnqxx6nxcLNfCFB4OL0RUinqtVj2LwVQCYi6vmnjJp5%2Fuc%2F%2FeJQDKrSA0Um9LcZaGiyjG7cQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C931240>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:09:28,805 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1320'}
2024-03-13 16:09:28,805 ERROR   wandb-upload_0:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:09:29,649 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:30,072 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:30,073 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:30,073 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:30,074 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:30,651 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:31,078 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:31,682 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:32,103 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:32,104 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:32,104 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:32,105 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:32,105 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:32,106 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:32,106 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:32,107 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:32,682 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:32,682 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:33,716 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:34,700 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:34,884 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:34,885 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:34,885 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:34,886 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:35,714 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:35,745 ERROR   gpu       :27916 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 16:09:36,728 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:36,901 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:36,904 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:36,904 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 16:09:36,905 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:36,906 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:36,906 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:36,907 DEBUG   SenderThread:27916 [sender.py:send():380] send: history
2024-03-13 16:09:36,907 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: summary_record
2024-03-13 16:09:36,908 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:37,058 DEBUG   SenderThread:27916 [sender.py:send():380] send: exit
2024-03-13 16:09:37,058 INFO    SenderThread:27916 [sender.py:send_exit():585] handling exit code: 0
2024-03-13 16:09:37,058 INFO    SenderThread:27916 [sender.py:send_exit():587] handling runtime: 107
2024-03-13 16:09:37,059 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:37,060 INFO    SenderThread:27916 [sender.py:send_exit():593] send defer
2024-03-13 16:09:37,060 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,060 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 0
2024-03-13 16:09:37,060 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,060 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 0
2024-03-13 16:09:37,060 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 1
2024-03-13 16:09:37,060 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,060 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 1
2024-03-13 16:09:37,060 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,061 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 1
2024-03-13 16:09:37,061 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 2
2024-03-13 16:09:37,061 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,061 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 2
2024-03-13 16:09:37,061 INFO    HandlerThread:27916 [system_monitor.py:finish():190] Stopping system monitor
2024-03-13 16:09:37,061 DEBUG   SystemMonitor:27916 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2024-03-13 16:09:37,068 DEBUG   SystemMonitor:27916 [system_monitor.py:_start():170] Publishing last batch of metrics
2024-03-13 16:09:37,068 INFO    HandlerThread:27916 [interfaces.py:finish():202] Joined cpu monitor
2024-03-13 16:09:37,069 INFO    HandlerThread:27916 [interfaces.py:finish():202] Joined disk monitor
2024-03-13 16:09:37,110 INFO    HandlerThread:27916 [interfaces.py:finish():202] Joined gpu monitor
2024-03-13 16:09:37,110 INFO    HandlerThread:27916 [interfaces.py:finish():202] Joined memory monitor
2024-03-13 16:09:37,110 INFO    HandlerThread:27916 [interfaces.py:finish():202] Joined network monitor
2024-03-13 16:09:37,111 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,111 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 2
2024-03-13 16:09:37,111 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 3
2024-03-13 16:09:37,112 DEBUG   SenderThread:27916 [sender.py:send():380] send: stats
2024-03-13 16:09:37,112 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,112 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 3
2024-03-13 16:09:37,112 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,112 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 3
2024-03-13 16:09:37,112 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 4
2024-03-13 16:09:37,112 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,112 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 4
2024-03-13 16:09:37,113 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,113 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 4
2024-03-13 16:09:37,113 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 5
2024-03-13 16:09:37,113 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,113 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 5
2024-03-13 16:09:37,113 DEBUG   SenderThread:27916 [sender.py:send():380] send: summary
2024-03-13 16:09:37,114 INFO    SenderThread:27916 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 16:09:37,115 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,115 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 5
2024-03-13 16:09:37,115 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 6
2024-03-13 16:09:37,115 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,115 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 6
2024-03-13 16:09:37,115 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,115 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 6
2024-03-13 16:09:37,115 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 7
2024-03-13 16:09:37,115 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:37,116 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,116 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 7
2024-03-13 16:09:37,116 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,116 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 7
2024-03-13 16:09:37,741 INFO    Thread-12 :27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json
2024-03-13 16:09:37,975 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 8
2024-03-13 16:09:37,975 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,975 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 8
2024-03-13 16:09:37,975 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,975 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 8
2024-03-13 16:09:37,975 INFO    SenderThread:27916 [job_builder.py:build():281] Attempting to build job artifact
2024-03-13 16:09:37,976 INFO    SenderThread:27916 [job_builder.py:_get_source_type():404] no source found
2024-03-13 16:09:37,976 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 9
2024-03-13 16:09:37,976 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:37,976 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 9
2024-03-13 16:09:37,977 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:37,977 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 9
2024-03-13 16:09:37,977 INFO    SenderThread:27916 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-13 16:09:38,070 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:38,744 INFO    SenderThread:27916 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log
2024-03-13 16:09:38,745 INFO    SenderThread:27916 [dir_watcher.py:finish():388] scan: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files
2024-03-13 16:09:38,745 INFO    SenderThread:27916 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\conda-environment.yaml conda-environment.yaml
2024-03-13 16:09:38,746 INFO    SenderThread:27916 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\config.yaml config.yaml
2024-03-13 16:09:38,748 INFO    SenderThread:27916 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\output.log output.log
2024-03-13 16:09:38,750 INFO    SenderThread:27916 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\requirements.txt requirements.txt
2024-03-13 16:09:38,753 INFO    SenderThread:27916 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-metadata.json wandb-metadata.json
2024-03-13 16:09:38,753 INFO    SenderThread:27916 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_160743-b3ojb8wd\files\wandb-summary.json wandb-summary.json
2024-03-13 16:09:38,755 INFO    SenderThread:27916 [sender.py:transition_state():613] send defer: 10
2024-03-13 16:09:38,755 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:38,755 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: defer
2024-03-13 16:09:38,756 INFO    HandlerThread:27916 [handler.py:handle_request_defer():170] handle defer: 10
2024-03-13 16:09:38,758 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: defer
2024-03-13 16:09:38,758 INFO    SenderThread:27916 [sender.py:send_request_defer():609] handle sender defer: 10
2024-03-13 16:09:38,758 INFO    SenderThread:27916 [file_pusher.py:finish():175] shutting down file pusher
2024-03-13 16:09:39,074 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:39,075 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:40,080 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:40,081 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:41,088 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:41,089 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:42,097 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:42,097 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:43,110 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:43,110 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:44,120 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:44,120 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:45,132 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:45,133 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:46,140 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:46,141 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:47,152 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:47,152 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:47,152 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:48,160 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:48,160 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:49,173 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:49,174 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:50,178 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:50,178 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:50,997 ERROR   wandb-upload_2:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/config.yaml?Expires=1710403781&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=wnhlVhLZJPDBcGdl0Ak%2BBbpCULsnSAYI8ZY0fbGvmt2MVDhTUsy0OENaUa977DxD30A6vodywuEXInzkE8d6UfxzpErw6tzQgNm3f12IdS%2BA8ZJ1ziTH3Tju418xLeDe8AUfzsFQ4Cag0fQcyqjj5CB99ibyIjefUmspVopr%2F9lOJU6AschdYNvRu9SKu1gLq9erQGUApXxd%2B5nRV%2BVuJnmYl2NGFCF9KBsgDJtflYgihPNHPbqRAn2jkXhTVL4%2FZoSvLI6NQb7zH905QoWJtNgkNgkUe4UpNSrPPWQbLI5qOBYGw5p64tIyoNoM0HNtvFYUpFnh%2B4pg%2Bzgvd%2Bdz1Q%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/config.yaml?Expires=1710403781&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=wnhlVhLZJPDBcGdl0Ak%2BBbpCULsnSAYI8ZY0fbGvmt2MVDhTUsy0OENaUa977DxD30A6vodywuEXInzkE8d6UfxzpErw6tzQgNm3f12IdS%2BA8ZJ1ziTH3Tju418xLeDe8AUfzsFQ4Cag0fQcyqjj5CB99ibyIjefUmspVopr%2F9lOJU6AschdYNvRu9SKu1gLq9erQGUApXxd%2B5nRV%2BVuJnmYl2NGFCF9KBsgDJtflYgihPNHPbqRAn2jkXhTVL4%2FZoSvLI6NQb7zH905QoWJtNgkNgkUe4UpNSrPPWQbLI5qOBYGw5p64tIyoNoM0HNtvFYUpFnh%2B4pg%2Bzgvd%2Bdz1Q%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C932590>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:09:50,997 ERROR   wandb-upload_2:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '456'}
2024-03-13 16:09:50,997 ERROR   wandb-upload_2:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:09:51,002 ERROR   wandb-upload_1:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/conda-environment.yaml?Expires=1710403781&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ElK3IBjkp62Ai3G9RGnzAHMpn0RZ0bVhnsVcXmjMNl49BQdUe8QYsy0SN4uRhpNxIVTjKNeGmkoD3QruZEs8NtRkWxgprKNyyvhvLh%2BNuHRvmQrdxGVqKwz7DxsussPK4lPrk61uRdYDFIeXGWb6ydd7EOV8NaS5vkPiSm5kgQiRdWBvDowkXjh84Flwt3lmPymSWYbQtypIyHH130UkjXuYKuEyLUuIAMAnlZoTH9z4W1uHd82woEcKVtH5%2Fkrvzyq5gRaK0UpirZEo5bFJY%2B6MbWBxm1d3Obfnr0JEJXQ%2Fz6Dex7q%2F%2BXUVP7oS8pXCXk0KjCMcUoGXbjkG0tR4Yw%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/conda-environment.yaml?Expires=1710403781&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ElK3IBjkp62Ai3G9RGnzAHMpn0RZ0bVhnsVcXmjMNl49BQdUe8QYsy0SN4uRhpNxIVTjKNeGmkoD3QruZEs8NtRkWxgprKNyyvhvLh%2BNuHRvmQrdxGVqKwz7DxsussPK4lPrk61uRdYDFIeXGWb6ydd7EOV8NaS5vkPiSm5kgQiRdWBvDowkXjh84Flwt3lmPymSWYbQtypIyHH130UkjXuYKuEyLUuIAMAnlZoTH9z4W1uHd82woEcKVtH5%2Fkrvzyq5gRaK0UpirZEo5bFJY%2B6MbWBxm1d3Obfnr0JEJXQ%2Fz6Dex7q%2F%2BXUVP7oS8pXCXk0KjCMcUoGXbjkG0tR4Yw%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C771660>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:09:51,002 ERROR   wandb-upload_1:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9492'}
2024-03-13 16:09:51,002 ERROR   wandb-upload_1:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:09:51,181 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:51,182 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:51,597 ERROR   wandb-upload_3:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/output.log?Expires=1710403782&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=sQFL3SbLq4sPYgzdgJyAFEuYhOVTfDuYRZOd%2BUAeZYCOFPzcPl%2FI%2FE1JI3JAV6kNkusTp0mR99BtP9%2FTsFfcCR15FptCL4U9nYTQ2ZXpMCA473yvYmEDNpxJhG0Debm44S1oA77idGrk9xbjlNtcvZMb7YQgGFjDkz0hLxUxJJZyNztKsNXUm46VxjCCaFxSJmdmTz55eNFWLW2wv6BcXDh5E%2Fb1CCUolfDfMVzHyESn%2F3ig%2FkblZtilWV8F4faV%2FhlmB8bm1PdPs205ukP%2FwkSeXwQ7rMeeioTPTsV1lNSIozzwsR8vq9FHBHfbHgHHGPzqS0zZ1ZZxBIPv2ZD5lQ%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/output.log?Expires=1710403782&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=sQFL3SbLq4sPYgzdgJyAFEuYhOVTfDuYRZOd%2BUAeZYCOFPzcPl%2FI%2FE1JI3JAV6kNkusTp0mR99BtP9%2FTsFfcCR15FptCL4U9nYTQ2ZXpMCA473yvYmEDNpxJhG0Debm44S1oA77idGrk9xbjlNtcvZMb7YQgGFjDkz0hLxUxJJZyNztKsNXUm46VxjCCaFxSJmdmTz55eNFWLW2wv6BcXDh5E%2Fb1CCUolfDfMVzHyESn%2F3ig%2FkblZtilWV8F4faV%2FhlmB8bm1PdPs205ukP%2FwkSeXwQ7rMeeioTPTsV1lNSIozzwsR8vq9FHBHfbHgHHGPzqS0zZ1ZZxBIPv2ZD5lQ%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C932EF0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:09:51,597 ERROR   wandb-upload_5:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-summary.json?Expires=1710403782&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=lq7fr0oXeViHCcBCI%2FpCES2hkGIZwVHpPjwGieUfYNpUybObcdHygPCdr9TVc%2F67Am9INHSqjpcdqsbWRPUnU4ER4id3wWEKLSFHznjAI%2F16WcXUd2TVIiWJ7muOweK7pDONpTy18lUlAzPJ50m0FpuVVdj7%2BD4JXNAa%2FmM%2FCjWatgA3P6TYbEGWczRxTiYkobU57fRLVFDj4Vof0OX%2FhREWIDSb4x3F2FFn2cg3j87W9zxdMvv6K5iq43vHq0kvAfplYs4PTiQaTiZdr0gghfIztCUwwKaSvcpvDt2CwGTq30GERzNe2dGNp1h2dgdGrrY0LuRmFpndc%2FSV1As7Vw%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/wandb-summary.json?Expires=1710403782&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=lq7fr0oXeViHCcBCI%2FpCES2hkGIZwVHpPjwGieUfYNpUybObcdHygPCdr9TVc%2F67Am9INHSqjpcdqsbWRPUnU4ER4id3wWEKLSFHznjAI%2F16WcXUd2TVIiWJ7muOweK7pDONpTy18lUlAzPJ50m0FpuVVdj7%2BD4JXNAa%2FmM%2FCjWatgA3P6TYbEGWczRxTiYkobU57fRLVFDj4Vof0OX%2FhREWIDSb4x3F2FFn2cg3j87W9zxdMvv6K5iq43vHq0kvAfplYs4PTiQaTiZdr0gghfIztCUwwKaSvcpvDt2CwGTq30GERzNe2dGNp1h2dgdGrrY0LuRmFpndc%2FSV1As7Vw%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C933670>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:09:51,597 ERROR   wandb-upload_3:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1412'}
2024-03-13 16:09:51,597 ERROR   wandb-upload_5:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '210'}
2024-03-13 16:09:51,597 ERROR   wandb-upload_3:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:09:51,598 ERROR   wandb-upload_5:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:09:51,601 ERROR   wandb-upload_4:27916 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/requirements.txt?Expires=1710403782&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=uo7b3LgHtdIuAGJM%2FNtAk8V72fcpjfuuKTCBGiYRw9Y7uS78kNp7r1W6SLOEeXE%2FZYn4HRfWslCkydngfMDjH0iGnuWo36msakE9JJByZLWzmx1OLjQGWUme0oUUZiwPuc4MuG5xaBmkD2YH4VrhcnCklDxiiCCSCkxf9a7TKWu6XD6asEkV1Jx%2Bjj2kUYqQ%2BzBkKI4RBVtKhacng57XRBfb%2BGEcWebH64GYpqUBVS9YmkuW3VLzrV077Bd%2B5D5h4pRQTxy5bxOSTCvoZPRvKwR9j9SR%2B2KNu5rgawEBIv4iLo5lnkCmLoCAdjddjYruL82QPTcLePLTJO2fjAmLPg%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/b3ojb8wd/requirements.txt?Expires=1710403782&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=uo7b3LgHtdIuAGJM%2FNtAk8V72fcpjfuuKTCBGiYRw9Y7uS78kNp7r1W6SLOEeXE%2FZYn4HRfWslCkydngfMDjH0iGnuWo36msakE9JJByZLWzmx1OLjQGWUme0oUUZiwPuc4MuG5xaBmkD2YH4VrhcnCklDxiiCCSCkxf9a7TKWu6XD6asEkV1Jx%2Bjj2kUYqQ%2BzBkKI4RBVtKhacng57XRBfb%2BGEcWebH64GYpqUBVS9YmkuW3VLzrV077Bd%2B5D5h4pRQTxy5bxOSTCvoZPRvKwR9j9SR%2B2KNu5rgawEBIv4iLo5lnkCmLoCAdjddjYruL82QPTcLePLTJO2fjAmLPg%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002240C933970>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 16:09:51,601 ERROR   wandb-upload_4:27916 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3463'}
2024-03-13 16:09:51,601 ERROR   wandb-upload_4:27916 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 16:09:52,191 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:52,191 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:52,192 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:53,199 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:53,200 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:54,212 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:54,213 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:55,226 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:55,226 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:56,241 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:56,241 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:57,242 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:09:57,242 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:57,243 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:58,244 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:58,244 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:09:59,252 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:09:59,253 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:10:00,267 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:10:00,268 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:10:01,279 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:10:01,279 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:10:02,290 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:10:02,291 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 16:10:02,291 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:10:03,299 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:10:03,300 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:10:04,308 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:10:04,308 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:10:05,316 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:10:05,316 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 16:10:06,317 DEBUG   HandlerThread:27916 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 16:10:06,317 DEBUG   SenderThread:27916 [sender.py:send_request():407] send_request: poll_exit
